apache/trafficcontrol
# TP: upgrades the version of nodejs that traffic portal runs on

## What does this PR (Pull Request) do?
`Node.js 6.x LTS Boron is no longer actively supported!`


## Which Traffic Control components are affected by this PR?

- Traffic Portal

## What is the best way to verify this PR?
- Run UI tests. Existing UI tests will be sufficient to prove the upgrade is good.
- Run TP in cdn-in-a-box and ensure that it works.

## The following criteria are ALL met by this PR
